docs/volunteering/glossary.md

```diff
@@ -0,0 +1,70 @@
+# Glossary
+
+While we try to use terminology which are clear and/or well known, here's a list
+you can check if you're not sure!
+
+<!---
+Note to editors; please:
+- keep this list in alphabetical order
+- ensure that, within the list, usages of other terms defined in the list are
+  emphasised suitably
+--->
+
+_Competition programme_:
+
+: the overall programme of events and activities which the teams participate in
+
+_Competition year_:
+
+: the 6 months (or so) between _Kickstart_ and the _Competition_ during which
+teams build their robots; see also _SRnnnn_
+
+_Core Team_:
+
+: the group of 8 or so volunteers responsible for delivering a particular year's
+_Competition programme_.
+
+[_GitHub_](https://github.com/srobo):
+
+: a platform for hosting code repositories which we also use to coordinate tasks
+
+_Kickstart_:
+
+: the start of the competition year, when we hand out kits to teams
+
+_Mentor_:
+
+: a volunteer who helps teams build their robots
+
+_Shepherd_:
+
+: a volunteer at the competition event who helps ensure teams make it to their
+matches on time
+
+[_Slack_](https://studentrobotics.slack.com/):
+
+: an instant-messaging tool we use for quick discussions
+
+_SR_:
+
+: a shortened form of Student Robotics
+
+[_SRComp_](https://github.com/PeterJCLaw/srcomp/wiki):
+
+: a software stack which we use to run competition events
+
+_srobo_:
+
+: another shortened form of Student Robotics
```

Comment:
Same as above, plus a mention of SR above
```suggestion
: another shortening of 'Student Robotics' (same as _SR_)
```

Reply:
40d6595f7a18790e96b6ad8ec0d4566e96abbdb4